### ISSUE TITLE
Missing shortcut to quikly check "Word Count"/"Document Info" panel

### ISSUE THREAD

**Issue by uncertainquark** [ONLYOFFICE/DesktopEditors#167]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Need to go to "File > Document Info" to see word count and other related stats.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Like MS Office, Google Docs and other office suites, assign a shortcut to check word count (and selected word count) and other related stats quickly. For example, see Google Docs shortcut of "Ctrl + Shift + C": https://techiestechguide.com/word-count-google-docs/

**What is the expected behavior?**
"Ctrl + Shift + C" should open "Word Count"/"Document Info" panel. 

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
v5.2.4

**Comment by Rita-Bubnova** [comment]
Hello, @uncertainquark.
We try to support keyboard shortcuts used in MS Office.
In your editor shortcut
Ctrl + Shift + C are used to copy formatting.

Currently we have no shortcut for opening info about document and created an issue. I created a new issue 40232 in our personal tracker about assigning a shortcut key to quickly view information about a document.
We will discuss this with our team.

**Comment by micah** [comment]
Hi @Rita-Bubnova -- has there been any progress on this?

**Comment by Medanna** [comment]
Hi, @uncertainquark 
It's planned to add word count anywhere closer than in doc info. Do you need qiuck access to doc info only because of word count or not?

**Comment by uncertainquark** [comment]
Need ability to see word count more easily than having to go to the doc into all the time.

**Comment by micah** [comment]
@Medanna Yes - it would be really good to be able to select a paragraph and get the word count, or character count of that selection. This is what people working on documents that have very specific requirements around word/character count need in order to meet these requirements (for example, grant writing). As it is now, we have to work on a paragraph in a desktop editor to get the count just right, and then paste it into OnlyOffice. Obviously, there is a higher margin of error with this flow, and  eats into the benefit of collaborative document editing if I am switching  between online and offline versions of the doc.

**Comment by uncertainquark** [comment]
To add to what @micah said, a quick display of word count for the selected text is also required for writers and journalists, where often we have limits on the number of characters/words in the title/subtitle.

**Comment by Medanna** [comment]
@micah @uncertainquark 
thank you. It's already planned, we will make it.

**Comment by uncertainquark** [comment]
Any update on this?

**Comment by Medanna** [comment]
@uncertainquark Hi, this feature is already designed and transferred to developers.

**Comment by knightfallx66** [comment]
Is there any indication when this will be released?

**Comment by Medanna** [comment]
Hi @knightfallx66 This feature has already been designed and is waiting for implementation.

**Comment by sinasartip** [comment]
Hi I was wondering if this has been released yet? I still dont see it, is there a setting that must be enabled?

**Comment by ShockwaveNN** [comment]
@sinasartip Sorry, no news. We'll comment at this bug as soon as we got some implementation

**Comment by fabioballi** [comment]
Hello. Any news? It seems quite long for an important function (academic texts, grant applications, etc must often respect a certain number of characters or words). Best,

**Comment by ShockwaveNN** [comment]
@fabioballi Sorry, no news. We'll comment at this bug as soon as we got some implementation

Our design team change focus to adding statistic panel to status bar, but it's still not implemented
There is issue 40224 in our private issue tracker

**Comment by KirillovIlya** [comment]
@fabioballi , we are working on it right now, you will see this new feature in one of our next releases.

**Comment by Batwam** [comment]
I still can't see it. Is this implemented yet? Also, could you please advise the macro command to add text to the status bar like `application.statusbar` in vba?

**Comment by TatianaKirillova** [comment]
Hello @Batwam! The problem with online playback remains and is indicated in bug 40224, but quick access to information is available using the button from the bottom menu Word count
